spec/features/cwic/granules/cwic_granule_filtering_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "spec_helper"
+
+describe "CWIC Granule list", reset: false do
```

Comment:
All acceptance criteria should have a matching spec that is tagged "acceptance: true"